### PR TITLE
 modified look and feel to make timeline work on autopsy-linux 

### PR DESCRIPTION
**Pull request by rishwanth1995**
lookandfeel is set to crossplatform instead of native look and feel.

### CODE REVIEW

File: Core/src/org/sleuthkit/autopsy/corecomponents/Installer.java [hunk 1]
```diff
@@ -111,4 +114,13 @@ private void setOSXLookAndFeel() {
             UIManager.put(entry.getKey(), entry.getValue());
         });
     }
+    
+    private void setUnixLookAndFeel(){
+        try {
+            //UIManager.setLookAndFeel(UIManager.getCrossPlatformLookAndFeelClassName());
```
Comment (rcordovano) [inline Core/src/org/sleuthkit/autopsy/corecomponents/Installer.java]: Why is there a duplicate line, commented out?

File: Core/src/org/sleuthkit/autopsy/corecomponents/Installer.java [hunk 2]
```diff
@@ -111,4 +114,13 @@ private void setOSXLookAndFeel() {
...
+            UIManager.setLookAndFeel(UIManager.getCrossPlatformLookAndFeelClassName());
+        } catch (ClassNotFoundException | InstantiationException | IllegalAccessException | UnsupportedLookAndFeelException ex) {
+            Exceptions.printStackTrace(ex);
```
Comment (rcordovano) [inline Core/src/org/sleuthkit/autopsy/corecomponents/Installer.java]: Please log the error, don't print it to stdout. 

What happens to the LAF if an exception is thrown here?

File: Core/src/org/sleuthkit/autopsy/corecomponents/Installer.java [hunk 3]
```diff
@@ -28,6 +28,7 @@
 import javax.swing.UnsupportedLookAndFeelException;
 import org.netbeans.swing.tabcontrol.plaf.DefaultTabbedContainerUI;
 import org.openide.modules.ModuleInstall;
+import org.openide.util.Exceptions;
```
Comment (rcordovano) [inline Core/src/org/sleuthkit/autopsy/corecomponents/Installer.java]: This import is not needed, see below.

File: Core/src/org/sleuthkit/autopsy/corecomponents/Installer.java [hunk 4]
```diff
@@ -40,6 +40,7 @@
     private static final long serialVersionUID = 1L;
     private static final Logger logger = Logger.getLogger(Installer.class.getName());
     private static Installer instance;
+    public static boolean enableTimeline = true;
```
Comment (rcordovano) [inline Core/src/org/sleuthkit/autopsy/corecomponents/Installer.java]: Our coding guidelines prohibit global variables. Please put this value in a properties file for Timeline to read. There is a ModuleSettings utility class that could be used for managing the properties file.

Note that you will need to reset the property to enabled at the beginning of this set LAF operation.

File: Core/src/org/sleuthkit/autopsy/corecomponents/Installer.java [hunk 5]
```diff
@@ -111,4 +114,18 @@ private void setOSXLookAndFeel() {
...
+        try {
+            UIManager.setLookAndFeel(UIManager.getCrossPlatformLookAndFeelClassName());
+        } catch (ClassNotFoundException | InstantiationException | IllegalAccessException | UnsupportedLookAndFeelException ex) {
+            logger.log(Level.WARNING, "Error setting crossplatform look-and-feel, setting default look-and-feel",ex);
+            try {
+                UIManager.setLookAndFeel(UIManager.getSystemLookAndFeelClassName());
+                enableTimeline=false;
+            } catch (ClassNotFoundException | InstantiationException | IllegalAccessException | UnsupportedLookAndFeelException ex1) {
+                 logger.log(Level.WARNING, "Error setting native look-and-feel",ex1);
+            }
```
Comment (rcordovano) [inline Core/src/org/sleuthkit/autopsy/corecomponents/Installer.java]: Given that you say that the default system LAF will be used anyway, I think we should reduce code complexity here (i.e., eliminate the nested try-catch) and simply set the property to false after logging the error.

File: Core/src/org/sleuthkit/autopsy/corecomponents/Installer.java (hunk 2 above)
Comment (bcarrier) [reply Core/src/org/sleuthkit/autopsy/corecomponents/Installer.java]: I think we can solve that in the next release.  This is good enough for now.

**Pull request merged**